projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -105,11 +121,11 @@ public static function enqueue_stats_script() {
 	/**
 	 * Gets the stats footer for AMP output.
 	 *
-	 * @access private
+	 * @access public
 	 * @param array $data Array of data for the AMP pixel tracker.
 	 * @return string Returns the footer to add for the Stats tracker in an AMP scenario.
 	 */
-	private static function get_amp_footer( $data ) {
+	public static function get_amp_footer( $data ) {
```

Comment:
Why are you changing this to be public?

If it's just to call it from tests, you could use `ReflectionMethod` to access it instead of having to make it public, or use a wrapper like [wikimedia/testing-access-wrapper](https://packagist.org/packages/wikimedia/testing-access-wrapper) like we do in packages/access and packages/changelogger.